src/utils/text.js

```diff
@@ -51,7 +51,8 @@ export const revenueLabelText = 'dollars'
 
 export const subscriberMinutesLabelText = 'minutes watched'
 
-export const noRevenueDescriptionText = `There's no revenue to report. Publishing lessons is the key.`
+export const noRevenueDescriptionText = `There's no stats to report. Publishing lessons is the key.`
```

Comment:
this would technically be "There are" 🤓  so maybe something like "No stats to report! Publishing lessons is the key."